aidial_assistant/chain/history.py

```diff
@@ -91,3 +123,63 @@ def to_best_effort_messages(
         )
 
         return messages
+
+    async def trim(
+        self, max_prompt_tokens: int, model_client: ModelClient
+    ) -> "History":
+        extra_results_callback = ModelExtraResultsCallback()
+        stream = model_client.agenerate(
```

Comment:
This will be replaced with a call to the model `history_trimming/` endpoint once we support it. Right? Maybe add a comment.

Reply:
Right. Added comment.